tiledb/sm/fragment/fragment_metadata.cc

```diff
@@ -827,6 +829,10 @@ Status FragmentMetadata::get_footer_size(
   return Status::Ok();
 }
 
+uint64_t FragmentMetadata::get_footer_size() const {
```

Comment:
For getters that do not return `Status` and instead return the requested value, I typically remove `get_`. I suggest naming this functions `footer_size()`.